### ISSUE TITLE
Font issues

### ISSUE THREAD

**Issue by RaxaStudios** [RaxaStudios/kfbot2.0#14]
On load, certain settings are not kept or applied with the font. Namely upon opening the Commands page then switching to the editor tab or the repeating tab, the font is reset to default and will change when coming in and out of focus by clicking on elements and then onto the backdrop. Other pages don't respect certain things like size and bold settings.

Possible fixes: CSS file addition and pulling, configuring each page closer with setting font, seeing if the bold/cursive/etc is having issues with the Comfortaa font being imported and used. 

Font normalization issues occur on Commands.fmxl at runtime. Bold/font issues occur on all pages/tabs. Use Lottery.fmxl for comparison. Font blurriness may be related and occurs on the Commands.fxml ScrollPane commandList.

**Comment by RaxaStudios** [comment]
repeating command tab on opening: https://i.imgur.com/Drp9vN9.png
on clicking an element, in this case a button in the scrollpane: https://i.imgur.com/5FWR8UM.png

command editor comparison on settings versus actual: https://i.imgur.com/0vkt00R.png

command list page comparison on settings and blurry font issue: https://i.imgur.com/0KwQ6eB.png

**Comment by CarlCamit** [comment]
Yeah good idea, add a TODO to refactor so that the GUI part uses `.css` files to style instead of styling them as the component is built. Hopefully that fixes in the inconsistency because right now it's too difficult to determine how the scene graph is being built as the application interacts with other components.

**Comment by RaxaStudios** [comment]
UPDATE: stylesheet.css file and .ttf files have been added to source folder, fixed majority of the issue, still unable to set different fonts/weights to individual parts without issue
